_common/KeymanHosts.php

```diff
@@ -111,11 +117,18 @@ function __construct() {
         // Note: locally running sites via Docker require website-local-proxy to be running
         // Append reverse-proxy port
         $local_port = isset($env['KEYMAN_COM_PROXY_PORT']) ? ':'.$env['KEYMAN_COM_PROXY_PORT'] : '';
-        $this->s_keyman_com = "http://s.keyman.com.localhost{$local_port}";
-        $this->api_keyman_com = "http://api.keyman.com.localhost{$local_port}";
-        $this->help_keyman_com = "http://help.keyman.com.localhost{$local_port}";
-        $this->keyman_com = "http://keyman.com.localhost{$local_port}";
-        $this->keymanweb_com = "http://keymanweb.com.localhost{$local_port}";
+        $this->s_keyman_com = "http://s.keyman.com.localhost$local_port";
+        $this->api_keyman_com = "http://api.keyman.com.localhost$local_port";
+        $this->help_keyman_com = "http://help.keyman.com.localhost$local_port";
+        $this->keyman_com = "http://keyman.com.localhost$local_port";
+        $this->keymanweb_com = "http://keymanweb.com.localhost$local_port";
+
+        // These are needed for PHP-side access
+        $this->SERVER_s_keyman_com = "http://host.docker.internal:8054";
+        $this->SERVER_api_keyman_com = "http://host.docker.internal:8058";
+        $this->SERVER_help_keyman_com = "http://host.docker.internal:8055";
+        $this->SERVER_keyman_com = "http://host.docker.internal:8053";
+        $this->SERVER_keymanweb_com = "http://host.docker.internal:8057";
```

Comment:
Does this mean these port values are never overridden by KEYMAN_COM_PROXY_PORT?

And do they need to be manually kept in sync with [keyman-local-ports.inc.sh](https://github.com/keymanapp/shared-sites/blob/991627b4e5bbbcd5a9629d92c4d7de5300d20fe1/_common/keyman-local-ports.inc.sh#L4-L9)?

Reply:
Yes, if ports change, we'd need to fix both places (plus a couple of other places where this has snuck in)